ms-common/src/main/java/net/trajano/ms/common/Microservice.java

```diff
@@ -50,6 +51,14 @@
     public static void run(final Class<? extends Application> applicationClass,
         final String... args) {
 
+        System.setProperty("vertx.disableDnsResolver", "true");
+        System.setProperty("vertx.logger-delegate-factory-class-name", "io.vertx.core.logging.SLF4JLogDelegateFactory");
+
+        final File logbackFile = new File("logback.xml");
+        if (logbackFile.exists()) {
+            System.setProperty("logging.config", logbackFile.getAbsolutePath().toString());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') "getAbsolutePath" returns a string, there's no need to call "toString()". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1858)